src/daft-scan/src/lib.rs

```diff
@@ -378,6 +378,7 @@ pub struct ScanTask {
     pub metadata: Option<TableMetadata>,
     pub statistics: Option<TableStatistics>,
     pub file_path_column: Option<String>,
+    pub generated_fields: IndexMap<std::string::String, Field>,
```

Comment:
I think we should be able to remove file_path_column from here and just keep generated fields

Reply:
Sounds good, done